tasks/conf.yml

```diff
@@ -44,6 +44,12 @@
 - name: Set --http-port option
   template: src=http_port.j2 dest=/etc/marathon/conf/http_port
 
+- name: Set additional options
+  template: src=custom_option.j2 dest=/etc/marathon/conf/{{ item.name }}
```

Comment:
That was my first idea, but I used the template to be consistent with the existing role tasks ;)

ok, I will change to use `content` then!